tests/data/wf_repos/basic_format2_collection_wrong_assert/basic_format2.gxwf-test.yml

```diff
@@ -21,10 +21,10 @@
               text: Word
         second:
           asserts:
-            - that: has_text
```

Comment:
Does the version of this where these isn't a list - but it is just a dictionary keyed on the assertion types validate? That seems to be what we have documented in: https://planemo.readthedocs.io/en/latest/test_format.html#test-format. I assume yes but I wanted to make sure.

I definitely prefer the a flat list of dictionaries with the fixed ``that`` keyword to a list of nested dictionaries each with a single outer key. The structure of this feels very... awkward or forced. Is this really the syntax you prefer or is JSON schema coercing us in this direction?